growthstreet_zoo/zoo.py

```diff
@@ -41,58 +27,12 @@ def number_of_cages(self):
         """Return the number of cages in the zoo."""
         return len(self.cages)
 
-    def add_cage(self, cage):
+    def add_cage(self, name=None):
```

Comment:
I think it would be a nicer API if you just provided the cage object, not a name for the cage object.